Automattic/woocommerce-payments
# Improved design of Payment Settings Page in mobile view

Fixes #6428 

#### Changes proposed in this Pull Request

- Aligned mobile view changes for additional payment method

Please refer to [this design](C1r0Vzxh1mhbDJJnbjHp5q-fi-2135_82787) for expected design

### Layout 2

| Before | After |
|--------|--------|
| ![image (3)](https://github.com/Automattic/woocommerce-payments/assets/110893979/fd26f540-bb30-4d15-8962-40801b951039) | ![image](https://github.com/Automattic/woocommerce-payments/assets/15019298/39084364-de62-40ec-b490-041e7c2b8ea2) |









#### Testing instructions





- Goto Payment > Settings
- Enable Multiple payment methods by switching store currency to EUR
- Open the browser dev tools and switch to the mobile view
- Check mobile responsiveness as expected
- Make sure the desktop version UI doesn't change and is as expected.
- Check if the functionality is intact and doesn't change on either desktop or mobile.
- Please make sure changes of PR https://github.com/Automattic/woocommerce-payments/pull/6875 are intact as there were merge conflicts


![image](https://github.com/Automattic/woocommerce-payments/assets/15019298/f09d4286-8431-4db2-bfdc-c7860df04c57)

-------------------


**Post merge**
